db/migrate/20200421154306_alter_admins.rb

```diff
@@ -0,0 +1,13 @@
+class AlterAdmins < ActiveRecord::Migration[6.0]
```

Comment:
try tu use more descriptive migration names like "change admin password column" or something like that